File: Backend/RoomMe/RoomMe.API/Models/ScheduleShortModel.cs
using System;
using System.Collections.Generic;
using System.Linq;
using System.Threading.Tasks;

namespace RoomMe.API.Models
{
    public class ScheduleShortModel
    {
        public int Id { get; set; }
        public DateTime Date { get; set; }
        public HouseworkStatusModel Status { get; set; }
        public UserNicknameModel User { get; set; }
    }
}

File: Android-app/domain/src/main/java/uj/roomme/domain/schedule/ScheduleShortModel.kt
package uj.roomme.domain.schedule

import uj.roomme.domain.housework.HouseworkStatusModel
import uj.roomme.domain.user.UserNicknameModel
import java.time.OffsetDateTime

data class ScheduleShortModel(
    val id: Int,
    val user: UserNicknameModel,
    val date: OffsetDateTime,
    val status: HouseworkStatusModel
)
